stripper/ze_v0u0v_csgo1.cfg

```diff
@@ -38,4 +38,35 @@ modify:
 	{
 		"OnStartTouch" "filter_zombieTestActivator01"
 	}
-}
\ No newline at end of file
+}
+
+;Fix kevlar damage not taken into account
+modify:
+{
+	match:
+	{
+		"classname" "trigger_hurt"
+	}
+	replace:
+	{
+		"damagetype" "0"
+	}
+}
+
+;Nerf ridiculous hp scaling
+modify:
+{
+	match:
+	{
+		"classname" "trigger_multiple"
+		"targetname" "s36miniboss_trigger"
+	}
+	delete:
+	{
+		"OnStartTouch" "s36miniboss_hitRunScriptCodeStart(MAP_STAGE==3 ? 280 : 360, 15);3.51"
+	}
+	insert:
+	{
+		"OnStartTouch" "s36miniboss_hitRunScriptCodeStart(MAP_STAGE==3 ? 100 : 180, 15);3.51"
+	}
+}
```

Comment:
I already told you I was looking into doing this change, I've already committed it to master so this needs to be removed.